src/routes/ConsortiumManager/settings/data-export/hooks/useDataExportLogs/useDataExportLogs.js

```diff
@@ -55,6 +57,7 @@ export const useDataExportLogs = (params = {}, options = {}) => {
     data,
   } = useQuery(
     [
+      DATA_EXPORT_LOGS_QUERY_KEY,
```

Comment:
Putting this in front of the namespace and tenant makes it easy to clear the cache up in `DataExportLogs.js`, but it also subverts much of the point of the query-key hierarchy. If you want to keep a shortcut around, could you export a `clearQueryCache()` function that encapsulates the query-key?